environments/gcp/flyte-core/iam.tf

```diff
@@ -1,3 +1,7 @@
+locals {
```

Comment:
Minor nit-pick:
I personally thick using underscores vs hyphens is more common in terraform.